core/storage/question/gae_models.py

```diff
@@ -244,6 +246,32 @@ def get_question_skill_links_and_skill_descriptions(
         )
         return question_skill_link_models, skill_descriptions, next_cursor_str
 
+    @classmethod
+    def get_question_skill_links_with_constant_number_per_skill(
```

Comment:
constant_number_per_skill is misleading because question_count may not divide evenly into len(skill_ids).

Maybe: get_question_skill_links_equidistributed_by_skill or similar perhaps? Gives the impression that there are N items and you are trying to spread them evenly.

Reply:
Done.